src/site/content/en/blog/web-platform-06-2022/index.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: "New to the web platform in June"
+subhead: >
+  Discover some of the interesting features that landed in stable and beta web browsers during June 2022. 
+description: >
+  Discover some of the interesting features that landed in stable and beta web browsers during June 2022.
+date: 2022-06-30
+hero: image/kheDArv5csY6rvQUJDbWRscckLr1/xPDclET3NIAnmeBygeTW.jpg
+alt: The word new on a red brick wall.
+authors:
+  - rachelandrew
+tags:
+  - blog
+  - new-to-the-web
+---
+
+## Stable browser releases
+
+In June, [Chrome 103](https://developer.chrome.com/blog/new-in-chrome-103/) and [Firefox 102](https://developer.mozilla.org/docs/Mozilla/Firefox/Releases/102) became stable.
+
+### Transform streams and readable byte streams
+
+Firefox 102 includes support for [Transform Streams](https://developer.mozilla.org/docs/Web/API/TransformStream). This enables piping from ReadableStream to a WritableStream, executing a transformation on the chunks. It's great to see this feature become available in all three engines, making this a very good time to learn about [Streams](/streams/).
+
+{% BrowserCompat 'api.TransformStream' %}
+
+[Readable byte streams](https://developer.mozilla.org/docs/Web/API/Streams_API#bytestream-related_interfaces) are also now supported in Firefox 102, enabling a BYOB (bring your own buffer) reader with the [ReadableStreamBYOBReader](https://developer.mozilla.org/docs/Web/API/ReadableStreamBYOBReader) interface. This can be used to stream data supplied by the developer. 
```

Comment:
```suggestion
[Readable byte streams](https://developer.mozilla.org/docs/Web/API/Streams_API#bytestream-related_interfaces) are also now supported in Firefox 102, enabling a BYOB (bring your own buffer) reader with the [`ReadableStreamBYOBReader`](https://developer.mozilla.org/docs/Web/API/ReadableStreamBYOBReader) interface. This can be used to stream data supplied by the developer. 
```